packages/neutron-sdk/src/bindings/msg.rs

```diff
@@ -15,6 +15,7 @@ pub struct IbcFee {
     pub recv_fee: Vec<Coin>,
     pub ack_fee: Vec<Coin>,
     pub timeout_fee: Vec<Coin>,
+    pub payer: Option<Addr>,
```

Comment:
It's a bit out of scope of the PR, but could you add some doc comments about each field like in the other messages in this file?
We somehow missed it during previous reviews

Reply:
Nevermind actually

I see some other things we should fix, let's make it in a separate PR